bot.py

```diff
@@ -862,6 +862,29 @@ def to_html():
     pass
 
 
+def split_json_file(data: dict, output_path: str, max_size: int = MAX_FILE_SIZE):
+    messages = data["messages"]
+    part_count = 1
+    current_data = {"messages": []}
+
+    for msg in messages:
+        current_data["messages"].append(msg)
+        serialized = json.dumps(current_data, default=str)
+        if len(serialized.encode('utf-8')) > max_size:
+            # Remove the last message that made it too big
+            current_data["messages"].pop()
+            # Save current part
+            with open(output_path.replace("result.json", f"result_part{part_count}.json"), "w") as outf:
```

Comment:
Also should change here too `f"result_part{page}`.

Reply:
not sure if I understand this